include/OpenSpaceToolkit/Astrodynamics/NumericalSolver.hpp

```diff
@@ -234,8 +253,10 @@ class NumericalSolver
         Real timeStep_ ;
         Real relativeTolerance_ ;
         Real absoluteTolerance_ ;
+        std::vector<StateVector> states_ ;
+        std::vector<double> times_ ;
```

Comment:
maybe call this `instants_` to be consistent with OSTk nomenclature?

Reply:
Good point. If it's not too heavy, I agree with this!

Reply:
fixed